<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintingUI</class>
 <widget class="QMainWindow" name="PaintingUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaintingUI</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPreviousView"/>
   <addaction name="actionNextView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Image Processing</string>
    </property>
    <addaction name="actionExtractColorVoxels"/>
    <addaction name="actionCreateImg4Projection"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPaintingOption">
    <property name="title">
     <string>Painting Option</string>
    </property>
    <addaction name="actionPenColor"/>
    <addaction name="actionPenWidth"/>
    <addaction name="separator"/>
    <addaction name="actionClearScreen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
   <addaction name="menuPaintingOption"/>
  </widget>
  <action name="actionmm">
   <property name="text">
    <string>mm</string>
   </property>
  </action>
  <action name="actionmj">
   <property name="text">
    <string>mj</string>
   </property>
  </action>
  <action name="actionNextView">
   <property name="text">
    <string>Next View</string>
   </property>
  </action>
  <action name="actionprevious">
   <property name="text">
    <string>previous</string>
   </property>
  </action>
  <action name="actionPreviousView">
   <property name="text">
    <string>Previous View</string>
   </property>
  </action>
  <action name="actionExtractColorVoxels">
   <property name="text">
    <string>Extract Color Voxels</string>
   </property>
  </action>
  <action name="actionCreateImage4Projection">
   <property name="text">
    <string>Create Image for Projection</string>
   </property>
  </action>
  <action name="actionCreateImg4Projection">
   <property name="text">
    <string>Create Image for Projection</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionPenColor">
   <property name="text">
    <string>Pen Color</string>
   </property>
  </action>
  <action name="actionPenWidth">
   <property name="text">
    <string>Pen Width</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionClearScreen">
   <property name="text">
    <string>Clear Screen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PaintingUI.qrc"/>
 </resources>
 <connections/>
</ui>
